uint8 ACTION_AVAILABLE=0
uint8 ACTION_USING=1
uint8 ACTION_RELEASE=2

uint8 action
string path
string node_pid
